.github/workflows/qa.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
             - name: Install PHP with extensions
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: 7.4
+                  php-version: 8.0
```

Comment:
Are we sure about this? IMO we should stick to `7.4` or try to run it in both versions.

Reply:
This is just a merge, the change was not made here but in 3.x
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/.github/workflows/qa.yaml#L30

It was automatically made by the dev-kit
https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/qa.yaml.twig#L32
which is by default the latest php version.
https://github.com/sonata-project/dev-kit/blob/3ce0e91eb8f5c0e70fa486e5e15e6703c4ea28bf/src/Domain/Value/Branch.php#L62

Reply:
Yeah I know, just pointing this out when I've seen it.

Reply:
We only ran static analysis on the last supported php version.

If we start running on multiple version, we could have issue like 
```
if (false === file_get_contents('foo')) {}
```
reported as useless in PHP8 but not in PHP7.

In this case, the right version might be the lowest one in a general way.
Do you want to open an issue or a PR on dev-kit ?

I'd like to avoid to delay this huge merge.